server/controllers/frameController.js

```diff
@@ -0,0 +1,35 @@
+import { Frame } from "../models/index.js";
+import { NotFound } from "../utils/httpError.js";
+
+export const getAllFrames = async (req, res) => {
+  const frames = await Frame.find();
+  res.status(200).json(frames);
+  if (!frames) {
+    throw new NotFound("No frames found");
+  }
```

Comment:
This is not a recommended practice. Why do you send the results before checking their validity? If there's an error, isn't it better to throw it instantly and halt the process instead of sending an `undefined`?